locale/en.po

```diff
@@ -6040,22 +5691,29 @@ msgstr ""
 "\n"
 "Follow the line from your cell to nearby glucose."
 
-#: ../src/tutorial/microbe_stage/MicrobeTutorialGUI.tscn:202
+#: ../src/tutorial/microbe_stage/MicrobeTutorialGUI.tscn:205
+msgid "MICROBE_STAGE_DAY_NIGHT_TEXT"
+msgstr ""
+"When the screen darkens it means night has come.\n"
+"During night your thylakoids and chloroplasts stop producing glucose from photosynthesis. \n"
+"Make sure you have enough storage and resources to survive the night when using those organelles."
```

Comment:
Other people might have a better idea but I think the phrasing could still be improved. I suggest adding a reference to the daylight meter in the environment panel for one.

Reply:
I'll also vote that the wording should be improved. For one the screen doesn't darken, but the environment does. The text can be made shorter by saying that photosynthesis is ineffective during the night. And the last part "when using those organelles" could be omitted. Instead these changes probably add enough space to refer to the environment panel saying the player can look there for the light level.